blackbox/docs/sql/reference/create_function.txt

```diff
@@ -0,0 +1,55 @@
+.. highlight:: psql
+.. _ref-create-function:
+
+===============
+CREATE FUNCTION
+===============
+
+Create a new function.
+
+.. warning::
+
+    There are currently no languages for user defined functions available!
```

Comment:
this needs clarifying. does this mean the functions can't be created at all? that the language param can't be used? is there a default? etc etc